<ui version="4.0" >
 <class>SettingsTab</class>
 <widget class="QFrame" name="SettingsTab" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Settings</string>
  </property>
  <property name="frameShape" >
   <enum>QFrame::NoFrame</enum>
  </property>
  <property name="frameShadow" >
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" >
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QGroupBox" name="groupBox" >
       <property name="title" >
        <string>Cpu / Memory</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3" >
        <item>
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Number of virtual CPUs:</string>
          </property>
          <property name="buddy" >
           <cstring>cpuSpinBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" >
          <item>
           <widget class="QSpinBox" name="cpuSpinBox" />
          </item>
          <item>
           <widget class="QLabel" name="label" >
            <property name="text" >
             <string>Virtual CPU(s)</string>
            </property>
            <property name="buddy" >
             <cstring>cpuSpinBox</cstring>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>Memory for this virtual machine:</string>
          </property>
          <property name="buddy" >
           <cstring>memorySpinBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" >
          <item>
           <widget class="QSlider" name="memorySlider" >
            <property name="maximum" >
             <number>1024</number>
            </property>
            <property name="singleStep" >
             <number>16</number>
            </property>
            <property name="pageStep" >
             <number>128</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="memorySpinBox" >
            <property name="maximum" >
             <number>1024</number>
            </property>
            <property name="singleStep" >
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="virtCheckBox" >
          <property name="text" >
           <string>Enable Virtualization</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2" >
       <property name="title" >
        <string>Network</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string>Usb</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_7" >
       <property name="title" >
        <string>Sound</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QGroupBox" name="groupBox_4" >
       <property name="title" >
        <string>Hard Disk</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_5" >
       <property name="title" >
        <string>Removable Media</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_6" >
       <property name="title" >
        <string>Additional</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cpuSpinBox</tabstop>
  <tabstop>memorySlider</tabstop>
  <tabstop>memorySpinBox</tabstop>
  <tabstop>virtCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>memorySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>memorySpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>148</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel" >
     <x>305</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>memorySpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>memorySlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>305</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel" >
     <x>148</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
